### PR TITLE
Enable date/datetime to be writable in Homeassistant

### LINKED ISSUE #1120: MQTT Homeassistant Support for "datetime" and "date" Fields
### Description

Similar to https://github.com/john30/ebusd/issues/1006
I would like to request Homeassistant MQTT Discovery support for date(times)s aswell.

### PR DESCRIPTION
**Pull request by Xyaren**
Fixes https://github.com/john30/ebusd/issues/1120

### CODE REVIEW

File: contrib/etc/ebusd/mqtt-hassio.cfg [hunk 1]
```diff
@@ -147,10 +147,10 @@ circuit_part = {
 type_map-number = number
 type_map-list = string
 type_map-time = time
```
Comment (chrizzzp) [inline contrib/etc/ebusd/mqtt-hassio.cfg]: duplicate entry
`type_map-time = time`

Comment (Xyaren) [reply contrib/etc/ebusd/mqtt-hassio.cfg]: Thanks, fixed :)

**Review by chrizzzp — APPROVED** [review]

File: contrib/etc/ebusd/mqtt-hassio.cfg [hunk 2]
```diff
@@ -146,11 +146,10 @@ circuit_part = {
 # This is also an implicit field type filter as missing or empty mappings are not published at all.
 type_map-number = number
 type_map-list = string
+type_map-string = string
```
Comment (john30) [inline contrib/etc/ebusd/mqtt-hassio.cfg:149]: why is string included when the PR suggests addition of date/time values?

**Review by john30 — CHANGES_REQUESTED** [review]

File: contrib/etc/ebusd/mqtt-hassio.cfg (hunk 2 above)
Comment (Xyaren) [reply contrib/etc/ebusd/mqtt-hassio.cfg:149]: I don't remember. Feel free do remove it if it is not required.

**Pull request closed without merging**